polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { FindingType, FindingSeverity, Finding, getJsonRpcUrl, ethers, HandleBlock, Initialize } from "forta-agent";
+import { provideHandleBlock, provideInitialize } from "./agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test/index";
+import { Flag, POLYGON_VALIDATOR_SIGNER_ADDRESS, MINIMUM_THRESHOLD } from "./constants";
```

Comment:
```suggestion
import { Flag } from "./constants";
```
Suggestion to use mock _signed address_ and _minimum threshold_ rather than the same that are used by the bot. Therefore, these shouldn't be imported.